testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/singlenode-delta-lake-databricks/tempto-configuration.yaml

```diff
@@ -3,6 +3,7 @@ databases:
     jdbc_user: root
   delta:
     jdbc_driver_class: com.databricks.client.jdbc.Driver
+    jdbc_jar: https://repo1.maven.org/maven2/com/databricks/databricks-jdbc/2.6.32/databricks-jdbc-2.6.32.jar
```

Comment:
This will result in product test attempting to download the dependency at runtime, which may result in test flakiness.
We should download the dependency with maven, and thus utilize maven local cache.

Use `maven-dependency-plugin` `copy` goal
see eg `io.trino.tests.product.launcher.env.environment.EnvMultinodeConfluentKafka#KAFKA_PROTOBUF_PROVIDER`